jenkins/build.sh

```diff
@@ -43,6 +47,6 @@ done
 # exit code.  In this case, if the build command fails, the tee
 # command won't trick Jenkins into thinking the step passed.
 set -o pipefail
-npm --no-color run build -- --buildtype="$envName" --asset-source="$assetSource" --drupal-address="$drupalAddress" "$omitdebug" "$pullDrupal" 2>&1 | tee "$buildLog"
+npm --no-color run build -- --buildtype="$envName" --asset-source="$assetSource" --drupal-address="$drupalAddress" --drupal-max-parallel-requests="$drupalMaxParallelRequests" "$omitdebug" "$pullDrupal" 2>&1 | tee "$buildLog"
```

Comment:
Comment at the top 

```
# This script runs the vets-website build script.  The reason it's
# here is so the `set -o pipefail` will work properly; Jenkins gets
# mad when we use it in a sh command.
```

I don't what `set -o pipefail` does but it's important enough for this file to exist instead of Jenkins just calling the `yarn build` command directly?